src/include/daos/mem.h

```diff
@@ -288,6 +288,13 @@ enum {
 	UMEM_TYPE_ANY,
 };
 
+/* Hints for umem atomic copy operation primarily for bmem implementation */
+enum acopy_hint {
+	UMEM_COMMIT_IMMEDIATE = 0, /* commit immediate, do not call within a tx */
+	UMEM_COMMIT_DEFER,	/* OK to defer commit to blob to a later point */
+	UMEM_RESERVED_MEM	/* memory from dav_reserve(), commit on publish */
```

Comment:
The reserved region by default will be added in WAL redo log? I don't quite see the difference between DEFER & RESERVED_MEM...

Reply:
UMEM_COMMIT_DEFER => Update redo log, but do not commit it immediately to wal. Commit it when the next wal_commit/tx_end is called.
UMEM_RESERVED_MEM => Do not add the redo entry at all. This memory is reserved by umem_reserve() and umem_tx_publish() will create the associated redo entry.